android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabConfigHelper.kt

```diff
@@ -114,7 +115,11 @@ private fun SafeIntent.getColorExtra(name: String): Int? =
     if (hasExtra(name)) getIntExtra(name, 0) else null
 
 private fun getCloseButtonIcon(intent: SafeIntent, resources: Resources?): Bitmap? {
-    val icon = intent.getParcelableExtra(EXTRA_CLOSE_BUTTON_ICON) as? Bitmap
+    val icon = try {
+        intent.getParcelableExtra(EXTRA_CLOSE_BUTTON_ICON, Bitmap::class.java)
+    } catch (e: ClassCastException) {
```

Comment:
As discussed offline, it is leftover from a previous iteration where the method called did not have a checked cast, so it could, in theory, throw a `ClassCastException`. I think we should still keep it, if it is fine with you.